docs/01-app/02-building-your-application/02-data-fetching/01-fetching.mdx

```diff
@@ -561,7 +561,121 @@ export default async function Page({ params }) {
 }
 ```
 
-In addition, you can add a [Suspense Boundary](/docs/app/building-your-application/routing/loading-ui-and-streaming) to break up the rendering work and show part of the result as soon as possible.
+As below, you can add a [Suspense Boundary](/docs/app/building-your-application/routing/loading-ui-and-streaming) to break up the rendering work and show part of the result as soon as possible.
```

Comment:
```suggestion
Alternatively, you use [`<Suspense>`](https://react.dev/reference/react/Suspense) to break up the rendering work and stream albums and artists separately. Start by calling `getArtist` and `getAlbums` in their respective components. Then, wrap each component in its own Suspense boundary.
```

Reply:
Thanks for refining it :) applied to https://github.com/vercel/next.js/pull/73320/commits/ef83d2723f4f6d0ceba0ef53583699e198679b59